### ISSUE TITLE
BUG Not preserving endline after -- style comments, fails to compile

### ISSUE THREAD

**Issue by yevon** [liquibase/liquibase#2039]
## Environment

**Liquibase Version**:
4.3.3
**Liquibase Integration & Version**: <Pick one: CLI, maven, gradle, spring boot, servlet, etc.>
Oracle
**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**:
Oracle 19c
**Operating System Type & Version**:
Linux

## Description
The end line character after a -- comment is not preserved, and then packages fail to compile.

## Steps To Reproduce

This is an example of a changelog generated:

![image](https://user-images.githubusercontent.com/121157/129768539-79f45649-2521-42d5-b8ae-cda9e5fe4f00.png)

Which after applying the changelog results in:

![image](https://user-images.githubusercontent.com/121157/129768712-f3d0f85e-f97c-44aa-a2c3-a82396634a1e.png)

## Expected/Desired Behavior

Expected behaviour would be preserving endlines like exported file:

![image](https://user-images.githubusercontent.com/121157/129768850-2edfb090-edb5-42bd-b626-0509dda68d91.png)

**Comment by molivasdat** [comment]
Hi @yevon Thanks for writing up this issue. We will add this to the list of issues we process.

**Comment by yevon** [comment]
No, this happened with standalone liquibase library in version 4.3.3 when I reported it. I haven't tried again in latest liquibase release. I will try it again in next weeks in both liquibase standalone and sqlcl new versions and report the results.

**Comment by nvoxland** [comment]
Thanks, @yevon . Let me know what you find.

**Comment by yevon** [comment]
sqlcl 21.4 makes reference to this bug as fixed, I will try it again as now sqlcl gives support for sql files and gives more filter options. Now I don't have time to test this but I have to implement CI/CD for oracle in my company in the near future and I will have a look at it htne.

33489983 | LIQUIBASE UPDATE LOOSES BLANK LINES FOR PLSQL PACKAGES

**Comment by tati-qalified** [comment]
Hello @yevon, sorry for the late response.
The Liquibase version you're using is actually a forked version that's maintained by Oracle, so we're not able to help with this issue.
Here's the [Oracle support page](https://support.oracle.com/portal/) in case you're still having problems.

Thank you,
Tatiana
